extensions/github-review-requests/package.json

```diff
@@ -13,6 +16,13 @@
       "subtitle": "GitHub",
       "description": "Search review requests on GitHub and listing them.",
       "mode": "view"
+    },
+    {
+      "name": "actionablePullRequests",
+      "title": "Pending Pull Requests",
+      "description": "List actionable pending pull requests",
+      "mode": "menu-bar",
+      "interval": "1m"
```

Comment:
refreshing every minute feels a bit overenthusiastic 😅  what about using 5 / 10 / 15 minutes instead? 🤔

Reply:
It would be great if this value vas configurable from settings \*sigh\*

One GraphQL call with the given query burns 3 points. 60 calls in an hour result in 180 points. Github's resource limitation is [5000 points per hour](https://docs.github.com/en/graphql/overview/resource-limitations#rate-limit). Let's round it to 200: then, the extension would consume 4% of the available quota.

I would want to get updates as soon as they are available, although I understand the reasons you might give.

I'll be bargaining here a little bit: do 3 minutes still sound too much? Otherwise, I would go with 5 minutes.